### DISCUSSION TITLE
lazygit has started to fail

### DISCUSSION THREAD

**Discussion by danielo515** [LazyVim/LazyVim#896] [Q&A] [answered]
Hello.
I just updated lazyNvim to the latest version and suddently lazygit does not open anymore. It gives me the following error:
```

E5108: Error executing lua: ....local/share/Lazy/lazy/LazyVim/lua/lazyvim/util/init.lua:141: attempt to call method 'buf_valid' (a nil value)
stack traceback:
	....local/share/Lazy/lazy/LazyVim/lua/lazyvim/util/init.lua:141: in function 'float_term'
	...l/share/Lazy/lazy/LazyVim/lua/lazyvim/config/keymaps.lua:117: in function <...l/share/Lazy/lazy/LazyVim/lua/lazyvim/config/keymaps.lua:117>
E5108: Error executing lua: ....local/share/Lazy/lazy/LazyVim/lua/lazyvim/util/init.lua:141: attempt to call method 'buf_valid' (a nil value)
stack traceback:
	....local/share/Lazy/lazy/LazyVim/lua/lazyvim/util/init.lua:141: in function 'float_term'
	...l/share/Lazy/lazy/LazyVim/lua/lazyvim/config/keymaps.lua:117: in function <...l/share/Lazy/lazy/LazyVim/lua/lazyvim/config/keymaps.lua:117>

```

Interestingly, if I restart lazyvim it works once. So after opening then closing lazygit is what causes this to fail.

**Comment by folke** [comment, marked as answer]
You need to update lazy.nvim

**Reply by danielo515** [reply]
Thanks, that did the trick. Isn't a way to flag this kind of dependency updates? I try to update as less as possible, and this makes it really hard. How could I figure out which plugins need to be updated in tandem?

**Reply by folke** [reply]
you can't. For LazyVim, I expect both lazy.nvim and LazyVim to be updated together.
